web-ui/view/github.ml

```diff
@@ -327,6 +351,9 @@ let show_step ~org ~repo ~refs ~hash ~jobs ~variant ~job ~status ~csrf_token
         ]
       else []
     in
+    let step_info =
+      List.find_opt (fun (j : Client.job_info) -> j.variant = variant) jobs
+    in
```

Comment:
Just a suggestion (to follow ocaml.org code guideline) you can choose to apply or not:
```suggestion
    let step_info =
      let filter (j : Client.job_info) = j.variant = variant in
      List.find_opt filter jobs
    in
```